examples/classifier_example.py

```diff
@@ -3,25 +3,33 @@
 """
 
 from medinify.sentiment import ReviewClassifier
-# from medinify.sentiment import NeuralNetReviewClassifier
+from medinify.sentiment import NeuralNetReviewClassifier
 
 def main():
     """ Main function.
     """
 
     # Naive bayes classifier
```

Comment:
Can these examples be changed to what is best for new users? Looks like it's mostly leftovers from your testing right now. Using a file besides common-reviews would be good so it doesn't take so long.